File: app/src/main/java/com/bojie/weatherBo/util/UnitConvert.java
package com.bojie.weatherbo.util;

/**
 * Created by bojiejiang on 3/6/15.
 */
public
class UnitConvert {

    public static double fahrenheitToCelsius(double temperature) {
        temperature = ((temperature - 32) * 5.0) / 9.0;
        return temperature;
    }

    public static double celsiusToFahrenheit(double temperature) {
        temperature = (9.0 / 5.0) * temperature + 32;
        return temperature;
    }
}

File: app/src/main/java/com/bojie/weatherBo/util/UnitConvert.kt
package com.bojie.weatherbo.util

/**
 * Utility class for temperature unit conversion
 */
object UnitConvert {
    
    fun fahrenheitToCelsius(temperature: Double): Double {
        return ((temperature - 32) * 5.0) / 9.0
    }
    
    fun celsiusToFahrenheit(temperature: Double): Double {
        return (9.0 / 5.0) * temperature + 32
    }
} 